libcloud/compute/drivers/gig_g8.py

```diff
@@ -269,12 +273,14 @@ def ex_create_network(self, name, private_network="192.168.103.0/24",
                                     {"cloudspaceId": networkid})
         return self._to_network(network)
 
-    def ex_destroy_network(self, network):
+    def ex_destroy_network(self, ex_network):
```

Comment:
Sorry, I should have made this more clear in my original comment - ``ex_`` argument name prefix is only needed if the method name itself doesn't contain it.

If the method name already contains the prefix, then argument itself doesn't need to contain it.

I can address this when merging the PR.

Reply:
EDIT: I see you just pushed some more changes so perhaps you can address that and let me know when it's ready and I'll go ahead and merge it.